### ISSUE TITLE
Is anyone maintaining this project ?

### ISSUE THREAD

**Issue by dcgavril** [stephanenicolas/robospice#426]
Hello,
From what I can see there aren't any releases in a while now, can anyone tell me if we'll get any new release in the near future? (there are a lot of pull requests and issues reported, but as it seems, there isn't someone to approve them or to fix them)
Anyway, thanks for the hard work!

**Comment by softwaremaverick** [comment]
Yeah I don't mind doing it where I can. I'm trying to get back into doing Android development but haven't done it in a while.

There's definitely a few minor changes but which have a decent enough impact to be merged in.

I'll definitely be stringent on what goes in and what doesn't though ;-)

On 30 June 2015 15:16:00 BST, dcgavril <notifications@github.com> wrote:
>Hello,
>From what I can see there aren't any releases in a while now, can
>anyone tell me if we'll get any new release in the near future? (there
>are a lot of pull requests and issues reported, but as it seems, there
>isn't someone to approve them or to fix them)
>Anyway, thanks for the hard work!
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/stephanenicolas/robospice/issues/426

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by moxi** [comment]
I was thinking the same, I used this project last year, but I'm not so sure to use it in my new apps, there are a lot of open issues.

**Comment by stephanenicolas** [comment]
Hey guys,

I am one of the original authors. RoboSpice is not actively maintained,
that's sure. It's working well and as advertized ;) Nevertheless, since
it's development, I have more involved on the DI side with RoboGuice and
other projects related to reflection, byte code weaving and annotation
processing.

I agree that RoboSpice needs a refresh and I got some ideas, and a couple
good ideas have been submitted as well via PRs & issues. The problem is
that if RS remains a one man team project, or a team of a few people that
don't really have much time, it won't progress. Or progress will be slow.

We need more maintainers and I would be happy to add new people in. Send a
PR, ask to become a contributor, but definitely RS needs a more active
community of devs to change faster.

S.


2015-08-23 21:09 GMT-07:00 Roberto Carlos Gonzalez Flores <
notifications@github.com>:

**Comment by moxi** [comment]
II thought it was being maintained by a company, fair enough, I'll try to use it and help :)

Thanks for the library stephanenicolas

**Comment by stephanenicolas** [comment]
No, it has never really been maintained by a company. Only individual
contributors.

New contributors are welcome !

S.
